articles/bases.py

```diff
@@ -67,9 +67,8 @@ def get_urls(cls):
     def __str__(self):
         return self.title
 
-    @app_models.permalink
     def get_absolute_url(self):
-        return ('article_detail', 'articles.urls', (), {'slug': self.slug})
```

Comment:
Seems fine on the local server. Hmmm. I don't trust it but I can change it if there's some benefit?